controllers/tasks.js

```diff
@@ -123,10 +123,33 @@ const updateTask = async (req, res) => {
   }
 }
 
+/**
+ * Updates self task status
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const updateTaskStatus = async (req, res) => {
+  try {
+    const taskId = req.params.id
+    const { id } = req.userData
+    const task = await tasks.fetchSelfTask(taskId, id)
```

Comment:
`id` while being passed as function argument doesn't clearly tell which `id` it is
Lets rename it while it is being destructured to `userId` and then pass it ahead, would make much more sense then
https://wesbos.com/destructuring-renaming